### PR TITLE
Add a python script to execute ES request

### PR DESCRIPTION
**Pull request by sBouzols**

### CODE REVIEW

File: scripts/functions/indexes/elasticsearch.py [hunk 1]
```diff
@@ -7,10 +7,24 @@
...
 
+def prettyprint(result):
+    try:
+        pretty = dumps(result.json(), indent=2)
+        print(pretty)
+        return True
+    except JSONDecodeError as jsonE:
+        print('Response could not be JSON serialized')
+        print(result.text)
+        return True
```
Comment (antoinebhs) [inline scripts/functions/indexes/elasticsearch.py:26]: Why always True?

File: scripts/functions/indexes/elasticsearch.py [hunk 2]
```diff
@@ -38,6 +52,27 @@ def check_status_eleasticsearch(url):
...
+            print("An error occured : ")
+            prettyprint(result)
+            return False
+        print("-----------------------")
+        prettyprint(result)
+        return True
+    except JSONDecodeError:
+        print('Response could not be JSON serialized')
+        print(result.text)
+        return True
```
Comment (antoinebhs) [inline scripts/functions/indexes/elasticsearch.py:70]: Why true here?

File: scripts/ES_requests.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+print("\n")
+# Just getting an enlightening url opportunistically from here because it exists
+# TODO better ?
+plateformName = get_plateform_info()['redirect_uri']
+elasticsearch_host = get_eleasticsearch_host(constant.DIRECTORY_SERVER_HOSTNAME)
+# TODO don't parse here, instead have the server return structured information
+elasticsearch_ip = socket.gethostbyname(elasticsearch_host.split(':')[0])
+# TODO we force http but should get this protocol from the server, some servers are not exposed on http but only https for example
+elasticsearch_url = constant.HTTP_PROTOCOL + elasticsearch_host
```
Comment (antoinebhs) [inline scripts/ES_requests.py:45]: Can't this all be in a separate function as we duplicate from elsewhere ? would simplify this script

File: scripts/ES_requests.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+#
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+# Elastic REST API
+# cf. https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html
+
+parser = argparse.ArgumentParser(description='Send requests (REST APIs) to the ES server', )
+     
+parser.add_argument("-m", "--method", help="request method to execute on ES server", required=True)
```
Comment (anistouri) [inline scripts/ES_requests.py:27]:
```suggestion
parser.add_argument("-m", "--method", help="HTTP method to use for the request (e.g., GET, POST)", required=True)
```

File: scripts/functions/indexes/elasticsearch.py [hunk 5]
```diff
@@ -38,6 +52,27 @@ def check_status_eleasticsearch(url):
         print(e)
         return False
 
+def request_elasticsearch(method, url):
+    try:
+        print(method + " " + url)
+        result = requests.request(method, url)
+        if not result.ok :
+            # TODO this might not be a json format
```
Comment (antoinebhs) [inline scripts/functions/indexes/elasticsearch.py:60]: isn't it handled with the prettyprint()?

File: scripts/ES_requests.py [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+from functions.indexes.elasticsearch import check_status_eleasticsearch
+from functions.indexes.elasticsearch import request_elasticsearch
+from functions.plateform.plateform import get_plateform_info
+
+#
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+# Elastic REST API
+# cf. https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html
```
Comment (antoinebhs) [inline scripts/ES_requests.py:23]: Add an example maybe?
ES_requests.py -m get -u /_cat/indices
to get all the indices ?

File: scripts/functions/indexes/elasticsearch.py (hunk 2 above)
Comment (sBouzols) [reply scripts/functions/indexes/elasticsearch.py]: removed, done in `prettyprint`

File: scripts/functions/indexes/elasticsearch.py (hunk 1 above)
Comment (sBouzols) [reply scripts/functions/indexes/elasticsearch.py]: removed, it's a print function, we don't need a return

File: scripts/functions/indexes/elasticsearch.py (hunk 5 above)
Comment (sBouzols) [reply scripts/functions/indexes/elasticsearch.py]: true, comment moved in `prettyprint` Function

**Review by antoinebhs — APPROVED** [review]

**Review by anistouri — APPROVED** [review]

**Pull request merged**